### PR TITLE
DevTools 98 > Accessibility > Full-page view

### PR DESCRIPTION
**Pull request by sofiayem**
Documented https://developer.chrome.com/blog/new-in-devtools-98/#a11y-tree

### CODE REVIEW

File: site/en/docs/devtools/accessibility/reference/index.md [hunk 1]
```diff
@@ -125,12 +129,38 @@ To open the Accessibility pane:
...
 
-Inspect an element's position in the accessibility tree from [the Accessibility pane][8].
+Inspect an element's position in the accessibility tree from the [Accessibility pane][8].
 
-{% Img src="image/admin/IGM9xYMSTugsmo7sxIjk.png", alt="The Accessibility Tree section", width="800", height="506" %}
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/mTkkFUVuQ64mAY5iXCE3.png", alt="The Accessibility Tree section", width="800", height="524" %}
 
 **Figure 7**. The Accessibility Tree section
 
+This view allows you to explore only a single node and its ancestors.
```
Comment (jecfish) [inline site/en/docs/devtools/accessibility/reference/index.md:138]: expand the description

File: site/en/docs/devtools/accessibility/reference/index.md [hunk 2]
```diff
@@ -125,12 +129,38 @@ To open the Accessibility pane:
...
+
+### (Preview) Explore the accessibility tree {: #explore-tree}
```
Comment (jecfish) [inline site/en/docs/devtools/accessibility/reference/index.md:140]:
```suggestion
### (Preview) Explore the full-page accessibility tree {: #explore-tree}
```

File: site/en/docs/devtools/accessibility/reference/index.md [hunk 3]
```diff
@@ -3,13 +3,17 @@ layout: "layouts/doc-post.njk"
...
   - kaycebasques
+  - sofiayem
 date: 2018-07-13
 #updated: YYYY-MM-DD
 description: "A comprehensive reference of accessibility features in Chrome DevTools."
 tags:
   - accessibility
 ---
 
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons"
```
[the remark this answers is not in the document]
Comment (jeffposnick) [reply site/en/docs/devtools/accessibility/reference/index.md:14]: Is the only icon used that `<span class="material-icons">accessibility</span>`?

My understanding is that icon fonts are generally not a great idea, as you end up making several network requests and downloading unused bytes for something that could be accomplished via alternative means.

I would recommend downloading and using an inline SVG from https://fonts.google.com/icons?selected=Material+Icons&icon.query=accessibility instead, if possible.

**Review by jeffposnick — CHANGES_REQUESTED** [review]

File: site/en/docs/devtools/accessibility/reference/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sofiayem) [reply site/en/docs/devtools/accessibility/reference/index.md:14]: @jeffposnick Thank you for making me aware of that. (I was just following the [guide](https://developers.google.com/fonts/docs/material_icons#setup_method_1_using_via_google_fonts)). Will replace with an SVG shortly.

**Review by jeffposnick — APPROVED** [review]
Thanks for switching to SVGs!

**Pull request merged**